### DISCUSSION TITLE
[Enhancement] [date-picker] I hope that when selecting the date and year in the range, they can be chosen through a dropdown menu.

### DISCUSSION THREAD

**Discussion by daidaibg** [element-plus/element-plus#15582] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `date-picker`

## I hope that when selecting the date and year in the range, they can be chosen through a dropdown menu.

1."I hope to have the same functionality as tdesign, allowing for quick switching between years and months when selecting a range."

2."Add year range selection and quarter range selection."

[tdesign](https://tdesign.tencent.com/vue-next/components/date-picker)

### Additional comments

These requirements should contribute to a more optimized user experience and usability.

**Comment by chenxch** [comment]
Each component library has its own design style.
